catalog/nsili/catalog-nsili-sourcestoquery-ui/src/test/java/org/codice/alliance/catalog/nsili/sourcestoquery/ui/TestSourcesToQuery.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.nsili.sourcestoquery.ui;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.codice.alliance.catalog.nsili.sourcestoquery.ui.service.SourcesToQuery;
+import org.codice.alliance.nsili.endpoint.NsiliEndpoint;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.CatalogFramework;
+
+public class TestSourcesToQuery {
```

Comment:
Even though it hasn't been followed strictly, we are trying to name test classes FooTest instead of TestFoo.